mis_builder/models/mis_report_instance.py

```diff
@@ -459,6 +459,18 @@ def _check_source_cmpcol(self):
                         % rec.name
                     )
 
+    @api.returns(None, lambda value: value[0])
```

Comment:
The decorator is not necessary, it is inherited automatically (according to its documentation and examples in the Odoo codebase at least).

Reply:
Let's trust Odoo this time :)